src/main/java/uk/gov/pay/api/resources/SearchRefundsResource.java

```diff
@@ -66,17 +66,27 @@ public SearchRefundsResource(SearchRefundsService searchRefundsService, PublicAp
     )
     public SearchRefundsResults searchRefunds(@Parameter(hidden = true)
                                   @Auth Account account,
-                                  @Parameter(description = "From date of refunds to be searched (this date is inclusive). Example=2015-08-13T12:35:00Z")
+                                  @Parameter(description = "Returns refunds created on or after the `from_date`. " +
+                                          "Date and time must use Coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ssZ`.", example = "2015-08-13T12:35:00Z")
                                   @QueryParam("from_date") String fromDate,
-                                  @Parameter(description = "To date of refunds to be searched (this date is exclusive). Example=2015-08-14T12:35:00Z")
+                                  @Parameter(description = "Returns refunds created before the `to_date`. " +
+                                          "Date and time must use Coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ssZ`.", example = "2015-08-13T12:35:00Z")
                                   @QueryParam("to_date") String toDate,
-                                  @Parameter(description = "From settled date of refund to be searched (this date is inclusive). Example=2015-08-13")
```

Comment:
Can we add the `example` attribute here too?

Reply:
done